MAUI/ListView/swiping.md

```diff
@@ -0,0 +1,628 @@
+  favoriteImage.Source = ImageSource.FromResource("Swiping.Images.Favorites.png");
+  favoriteImage.BindingContextChanged += FavoriteImage_BindingContextChanged;
+  favoriteGrid.Children.Add(favoriteImage);
+  grid1.Children.Add(favoriteGrid);
+
+  var grid2 = new Grid()
+  {
+  BackgroundColor = Colors.FromHex("#DC595F"),
+  HorizontalOptions = LayoutOptions.Fill,
+  VerticalOptions = LayoutOptions.Fill
+  };
+  var deleteGrid = new Grid() { HorizontalOptions = LayoutOptions.Center, VerticalOptions = LayoutOptions.Center };
+  var deleteImage = new Image() { BackgroundColor = Colors.Transparent, HeightRequest = 35, WidthRequest = 35 };
+  deleteImage.Source = ImageSource.FromResource("Swiping.Images.Delete.png");
+  deleteImage.BindingContextChanged += DeleteImage_BindingContextChanged;
+  deleteGrid.Children.Add(deleteImage);
+  grid1.Children.Add(deleteGrid);
+
+  grid.Children.Add(grid1);
+  grid.Children.Add(grid2, 1, 0);
+                
+  return grid;
+});
+{%endhighlight %}
+{% endtabs %}
+
+![Customize Start swipe in listview](Images/swiping/maui-listview-working-with-multiple-views-in-swipetemplate.jpg)
+
+To delete the item when `Delete` image is tapped and setting favorites to item when `Favorites` image is tapped, follow the code example.
+
+{% tabs %}
+{% highlight c# %}
+Image leftImage;
+Image rightImage;
+int itemIndex = -1;
+
+private void SetFavorites()
+{
+   if (itemIndex >= 0)
+   {
+       var item = viewModel.InboxInfo[itemIndex];
+       item.IsFavorite = !item.IsFavorite;
+   }
+   this.listView.ResetSwipeItem();
+}
+
+private void Delete()
+{
+   if (itemIndex >= 0)
+       viewModel.InboxInfo.RemoveAt(itemIndex);
+   this.listView.ResetSwipeItem();
+}
+
+private void ListView_SwipeStarting(object sender, SwipeStartedEventArgs e)
+{
+   itemIndex = -1;
+}
+
+private void ListView_SwipeEnded(object sender, SwipeEndedEventArgs e)
+{
+   itemIndex = e.Index;
+}
+
+private void leftImage_BindingContextChanged(object sender, EventArgs e)
+{
+   if (leftImage == null)
+   {
+      leftImage = sender as Image;
+      (leftImage.Parent as View).GestureRecognizers.Add(new TapGestureRecognizer() { Command = new Command(SetFavorites) });
+      leftImage.Source = ImageSource.FromResource("Swiping.Images.Favorites.png");
+   }
+}
+
+private void rightImage_BindingContextChanged(object sender, EventArgs e)
+{
+   if (rightImage == null)
+   {
+      rightImage = sender as Image;
+      (rightImage.Parent as View).GestureRecognizers.Add(new TapGestureRecognizer() { Command = new Command(Delete) });
+      rightImage.Source = ImageSource.FromResource("Swiping.Images.Delete.png");
+   }
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Performing swipe delete operation
+
+To delete an item in view while swiping the item from one extent to other by using `SfListView.SwipeEnded` event. By setting the `SfListView.SwipeOffset` value to the view size to swipe the item upto end of the item.
```

Comment:
To delete an item in view while swiping the item from one extent to another, use the `SfListView.SwipeEnded` event. By setting the `SfListView.SwipeOffset` value to the view size to swipe the item up to the end of the item.

Reply:
resolved